source/transfer_list.rst

```diff
@@ -657,5 +658,86 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+.. _arm_entries:
+
+Arm specific entries
```

Comment:
I am open for this naming, let's see if there comes more inputs from @manish-pandey-arm @sjg20 @jwerner-chromium

Reply:
Agree to Dan's suggestion with minor nit "Trusted Firmware related entries". 

https://www.trustedfirmware.org/ has various projects (including TF-A, OP-TEE, TF-M, MCUboot etc). Even TF-M might use it in future. My preference is to avoid "-A", if it is usable for M class projects as well.

Reply:
Fixed.